picard/ui/savewarningdialog.py

```diff
@@ -35,24 +35,45 @@ def __init__(self, parent, file_count=None):
         config = get_config()
 
         if not config.setting['dont_write_tags']:
-            actions.append(_("overwrite existing metadata in the files"))
+            if file_count is None:  # Not provided -- use generic message
```

Comment:
In which cases `file_count` is `None`?

This is called once in https://github.com/metabrainz/picard/blob/2962cc641f9d93ad263c6fc8281243ea276571d7/picard/ui/mainwindow.py#L1356-L1357, and count comes from `len()` so it cannot be None.

IMHO this branch can be removed, having 3 translatable strings for each option isn't strictly needed anyway.

Reply:
You're right.  Looks like I got a bit carried away with the possibilities.  I'll remove those, which will reduce the number of strings to translate.